### ISSUE TITLE
iOS app not available in danish app stor 

### ISSUE THREAD

**Issue by NicolaiFriis** [phonegap/phonegap-app-desktop#883]
The app is not in the danish app store. But on android google play it is..

**Comment by chrisgriffith** [comment]
The iOS version has been removed due to a violation of Apple's Terms of Service. The ability to remote load radically different code via user input is the issue.

**Comment by NicolaiFriis** [comment]
Maybe they should update the getting started guide then: http://docs.phonegap.com/getting-started/2-install-mobile-app/
But thanks for the answer!

**Comment by NicolaiFriis** [comment]
@chrisgriffith do you know how to test the ipa then? 
Our app is build on a index.html file and when i preview that in chrome it looks correct. Now the app is updated in app store but now with some grey bars that was not there before. 
The changes i made to the app was only some links to the wordpress site which hostes content. Meaning all content is now collected from another website. 
The app is called Åben og Rolig and the files can be viewed here: https://github.com/NicolaiFriis/abenogrolig

**Comment by chrisgriffith** [comment]
Where are the grey bars?

Does it appear on all iOS devices or just the X?

I assume you don't have a Mac or Xcode to check it in the iPhone Sim?

Chris

**Comment by NicolaiFriis** [comment]
Top and bottom of the app. 
I fixed it with this viewport-fit=cover in the meta viewport in the index.html 

Im not sure how many devices had it. I only had a X for testing.

I have a Macbook and also Xcode but i have trouble with getting the app inside Xcode so i can preview it. When i build the app i get a ipa file from phonegap and that cant open i Xcode. 

I recently found out i could upload the build to apple and then use testflight to test the app on my iPhone X before i published it. 

But it would be great if you have a advice on getting the app inside Xcode so i could use iPhone Sim.

And thanks for all your time 👍
